.travis.yml

```diff
@@ -5,8 +5,8 @@ install:
 - pip install -r requirements.txt
 - ansible-galaxy install Datadog.datadog
 script:
-- ansible-playbook site.yml --check
-- ansible-playbook site_amazon.yml --check
+- ansible-playbook site_k8s.yml --syntax-check
```

Comment:
Why not use --check?

Reply:
Because check is failing on all amazon stuff. It doesn't really run so nothing is registered.